### DISCUSSION TITLE
Additional options to share content from browser

### DISCUSSION THREAD

**Discussion by wiesehahn** [community/community#14996] [Mobile]
Hi, e.g. to do research related to a project I like to use the tablet and the Github App.

In general I really like the option to use the share-button in the browser. However, at the moment the only option is to create a new issue which is not the desired action in many/most cases.

It would be really nice to have additional options. E.g. to add as comment to an existing issue or to some discussion. 

This would very much enhance the usability of the app when working from the couch :)

**Comment by stevepopovich** [comment]
Hey Jens! Thanks for the detailed feedback. Let me try and understand what you mean here...exactly what options are you looking for that the website has that the mobile app does not?

>  E.g. to add as comment to an existing issue or to some discussion.

The mobile app should support this. Maybe I am not fully understanding what you are looking for.

**Reply by wiesehahn** [reply]
Ok, I am not sure how this is implemented but I try to explain what I thought of.

In browsers like Firefox or Chrome but also in this app there is an option to share the current content with several apps (among others with Github). 

the symbol in the upper right corner
![Screenshot_20220425-192920_GitHub.jpg](https://user-images.githubusercontent.com/41429613/165142573-3893ce18-b376-4342-a290-2fb8afd3b061.jpg)


there I find one option related to Github which is to create a new issue 
![Screenshot_20220425-192945_Android System.jpg](https://user-images.githubusercontent.com/41429613/165143022-a2451dfa-8e53-469b-91e8-e963f2369302.jpg)

after selecting it I can choose a repository and optionally change title and content. 

What I want is the option to add the content (e.g. link, image) as a comment to an existing issue or an existing discussion. 

Does that answer your question?

**Reply by wiesehahn** [reply]
maybe something like just "GitHub" for this button, 
then choose the repository,
then choose between "create new issue", "create new discussion", "comment existing issue", "comment existing discussion"

**Reply by stevepopovich** [reply]
Gotcha, I see now. Sounds good, I will pass on to the team for prioritization!

**Reply by wiesehahn** [reply]
Any news on this?

**Reply by stevepopovich** [reply]
It is in our backlog, that is the best news I can give you unfortunately.
